homeassistant/components/camera/arlo.py

```diff
@@ -90,3 +94,35 @@ def model(self):
     def brand(self):
         """Camera brand."""
         return DEFAULT_BRAND
+
+    @property
+    def should_poll(self):
+        """Camera should poll periodically."""
+        return True
+
+    @property
+    def motion_detection_enabled(self):
+        """Camera Motion Detection Status."""
+        return self._motion_status
+
+    def set_base_station_mode(self, mode):
+        """Set the mode in the base station."""
+        self._base_stn.mode = mode
+
+    def async_enable_motion_detection(self):
```

Comment:
These methods need to be moved to the base entity with the sync version raising `NotImplementedError`. That's how we do it in other places in HASS too. See for example climate: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L608-L617